vyper/exceptions.py

```diff
@@ -151,6 +157,11 @@ def _add_hint(self, msg):
             return msg
         return msg + f"\n  (hint: {self.hint})"
 
+    def _format_contract_details(self, msg, path, lineno):
+        from vyper.utils import safe_relpath
```

Comment:
no need for this to be a local import -- it can go at the top of the file

Reply:
Its not at the top because of circular import

Reply:
ah right, forgot about that